docs/api.rst

```diff
@@ -255,6 +255,44 @@ or add a custom handler for ``sentinelsat`` (as implemented in ``cli.py``)
   h.setFormatter(fmt)
   logger.addHandler(h)
 
+More Examples
+-------------
+
+Search Sentinel 2 by tile
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To search for recent Sentinel 2 imagery by MGRS tile, use the `tileid` parameter:
+
+.. code-block:: python
+
+  from collections import OrderedDict
+  from sentinelsat import SentinelAPI
+
+  api = SentinelAPI('myuser', 'mypass')
```

Comment:
The rest of the docs use ('user', 'password') - would be nice if you could change that to be consistent.